MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -88,11 +88,15 @@ internal void SetVertexAttributeArray(bool[] attrs)
             }
         }
 
-        private void PlatformSetup()
+       private void PlatformSetup()
         {
 #if WINDOWS || LINUX || ANGLE
             GraphicsMode mode = GraphicsMode.Default;
-            var wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
+
+            var wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);
```

Comment:
> what do you think of setting up MonoGame WinForm Control and maybe Gtk Widget projects in the MonoGame repository?

If you mean setting up like a VS template project...  sure that could be useful.